packages/gestalt/src/RadioGroup.css

```diff
@@ -0,0 +1,3 @@
+.customClass {
```

Comment:
@ayeshakmaz Is this the default set up by the template? Remove and `run yarn run flow-generate:css` to remove 
packages/gestalt/src/RadioGroup.css.flow

Reply:
not a nit, Should be removed if unused